samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -598,6 +601,22 @@ This is only supported on the `Nordic nRF9160 DK`_ with an attached nRF7002 EK.
 
 See also :ref:`the paragraphs on the Wi-Fi location tracking method <nrf_cloud_mqtt_multi_service_wifi_location_tracking>`.
 
+Building with nRF Cloud logging support
+=======================================
+
+To enable `Zephyr Logging`_ to nRF Cloud using the :ref:`lib_nrf_cloud_log` library, add the following parameter to your build command:
+
+``-DOVERLAY_CONFIG=overlay_nrfcloud_logging.conf``
+
+This overlay allows the sample and various subsystems that have logging enabled to send their logs to nRF Cloud.
+Set the :kconfig:option:`CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL` option to the log level of messages to send to nRF Cloud, such as ``4`` for debug log messages.
+Set the :kconfig:option:`CONFIG_NRF_CLOUD_REST_DEVICE_MESSAGE_SAMPLE_LOG_LEVEL_DBG` option so that log messages are generated on each button press.
+
+By default, the overlay sends log messages to the cloud in JSON format.
```

Comment:
Is this true?

Reply:
I changed the nrfcloud log overlay in this PR from binary to JSON mode.

But the overlay is not used by default.  Is that what you thought I meant?